src/AppBundle/Service/UserLootManager.php

```
@@ -2,33 +2,50 @@
 
 namespace AppBundle\Service;
 
+use AppBundle\Entity\ClmAccount;
+use AppBundle\Entity\ClmRaid;
 use AppBundle\Repository\ClmAccountRepositoryInterface;
+use AppBundle\Repository\ClmRaidRepositoryInterface;
 
 class UserLootManager
 {
     /**
      * @var ClmAccountRepositoryInterface
      */
     private $accountRepository;
+    /**
+     * @var ClmRaidRepositoryInterface
+     */
+    private $raidRepository;
+
 
     /**
+     * UserLootManager constructor.
      * @param ClmAccountRepositoryInterface $accountRepository
+     * @param ClmRaidRepositoryInterface $raidRepository
      */
-    public function __construct(ClmAccountRepositoryInterface $accountRepository)
+    public function __construct(
+        ClmAccountRepositoryInterface $accountRepository,
+        ClmRaidRepositoryInterface $raidRepository
+    )
     {
        $this->accountRepository = $accountRepository;
+       $this->raidRepository = $raidRepository;
     }
 
     /**
      * @return \AppBundle\Entity\ClmAccount[]
      */
     public function getAllAccounts()
     {
-        $accounts = $this->accountRepository->findAll();
+        return $accounts = $this->accountRepository->findAll();
 
-        return $accounts;
     }
 
+    /**
+     * @param $id
+     * @return null| ClmAccount[]
```

Better throw an exception instead of returning ```null```.